tests/access-control.tests.ts

```diff
@@ -0,0 +1,257 @@
+import { consts, Roles } from './constants'
+import {bnArg, expect, fromSigner, setupContract} from './helpers'
+
+describe('MY_ACCESS_CONTROL', () => {
+  async function setup() {
+    return setupContract('my_access_control', 'new')
+  }
+
+  it('ERC 721 - mint works', async () => {
```

Comment:
Let's move ERC 721 tests to erc721.tests.ts file

Reply:
I prefer to not create a lot of files with suffixes. I think it better to have one file for one example. Inside of test you can you several description sections to separate tests for Erc71, AccessControl, and Both.